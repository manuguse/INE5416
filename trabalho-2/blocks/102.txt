1 1 1 9 9 9 9 13
2 2 2 10 10 13 9 13
3 6 7 7 7 13 13 13
3 3 8 8 8 14 14 14
4 4 4 4 8 15 15 16
5 4 4 4 12 15 15 16
5 5 5 11 12 11 15 15
5 5 5 11 11 11 15 17