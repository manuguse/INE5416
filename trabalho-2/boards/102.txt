- 1 - - 3 - - 5
- - - 1 - - 2 3
- - 2 - - - 6 -
- - 3 4 - - - -
4 7 - 6 1 5 - -
- - 1 - - - - -
6 - - - - 4 - 7
3 4 1 - 2 - - -