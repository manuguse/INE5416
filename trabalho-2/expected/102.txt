3 1 2 5 3 1 4 5
1 2 3 1 2 4 2 3
3 1 2 3 1 2 6 1
1 2 3 4 2 1 2 3
4 7 5 6 1 5 6 2
7 2 1 3 2 1 4 1
6 5 2 5 1 4 3 7
3 4 1 3 2 1 2 1